src/builds/builds.service.ts

```diff
@@ -1,7 +1,7 @@
 import { Injectable, HttpException, HttpStatus } from '@nestjs/common';
 import { CreateBuildDto } from './dto/build-create.dto';
 import { PrismaService } from '../prisma/prisma.service';
-import { Build, Project } from '@prisma/client';
+import { Build, Project, ProjectUpdateArgs, Subset } from '@prisma/client';
```

Comment:
Codacy found an issue: ['ProjectUpdateArgs' is defined but never used.](https://app.codacy.com/gh/Visual-Regression-Tracker/backend/pullRequest?prid=6300596)